docs/pages/operations-guide.md

```diff
@@ -279,71 +312,71 @@ $ dcos node ssh --master-proxy --leader "docker run mesosphere/janitor /janitor.
     -z dcos-service-$MY_SERVICE_NAME"
 ```
 
-# Diagnosis tools
+# Diagnostic Tools
 
-DC/OS clusters have several available tools which can be used to diagnose services which are running in that cluster. In addition to that, the SDK itself offers its own endpoints which describe what the Scheduler itself is doing at any given time.
+DC/OS clusters provide several tools for diagnosing services are running in the cluster. In addition, the SDK itself  its own endpoints, which describe what the Scheduler is doing at any given time.
```

Comment:
"diagnosing services _that_ are running"
"the SDK itself &lt;??> its own endpoints"